packages/visualizations/src/components/Format/LongTextFormat.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+    import { warn } from './utils';
+    import type { LongTextFormatProps } from './types';
+
+    type $$Props = LongTextFormatProps;
+
+    export let rawValue: $$Props['rawValue'];
+    export let display: $$Props['display'] = (v: string) => v;
+
+    $: format = (v: unknown) => {
+        if (typeof v !== 'string') {
+            warn(v, 'text');
+        }
+        if (display) {
+            return display(v as string);
+        }
+        return v;
+    };
+
+    $: value = format(rawValue);
```

Comment:
I think https://github.com/opendatasoft/ods-dataviz-sdk/pull/267 is the right place to implement that, since I generalize formatters there. But agreed on the general principle of inversion indeed 👌